### PR TITLE
Small fix to remove shifting of lm labels during pre process of RocStories.

### PR DESCRIPTION
**Pull request by apappu97**
In reference to https://github.com/huggingface/pytorch-pretrained-BERT/issues/473, remove the one shifting of lm labels since this shift happens internally during the model's forward pass. 

@thomwolf

### CODE REVIEW

**Comment by thomwolf** [comment]
Awesome, thanks!
